modules/importing-repositories/procedure-enabling-the-bulk-import-from-github-feature.adoc

```diff
@@ -0,0 +1,22 @@
+[id="enabling-ang-giving-access-to-the-bulk-import-feature"]
+= Enabling and giving access to the Bulk Import feature
+
+.Procedure
+. To give {product-short} access to GitHub, link:{linkgettingstartedguide}#configuring-github-integration[configure GitHub integration].
+
+. The Bulk Import plugin is installed but disabled by default.
+Enable the `./dynamic-plugins/dist/janus-idp-backstage-plugin-bulk-import-backend-dynamic` and `./dynamic-plugins/dist/janus-idp-backstage-plugin-bulk-import` preinstalled plugins.
```

Comment:
@rm3l https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.3/html-single/dynamic_plugins_reference/index#rhdh-tech-preview-plugins mentions only one plugin:

> Bulk Import
> @janus-idp/backstage-plugin-bulk-import
> 1.4.9
> ./dynamic-plugins/dist/janus-idp-backstage-plugin-bulk-import

Is the second plugin missing in the table, or has it been removed?

Reply:
The backend plugin is currently in the list of [community plugins](https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.3/html-single/dynamic_plugins_reference/index#rhdh-community-plugins).
This was a mistake that is already fixed in https://github.com/janus-idp/backstage-plugins/pull/2223 (but for 1.4 (where both plugins will have the same support level - Tech Preview)